ledger-core/insolar/payload/validation.go

```diff
@@ -22,6 +22,14 @@ func isSpecialTimePulseBefore(pn pulse.Number, before pulse.Number) bool {
 	return pn.IsSpecial() || pn.IsTimePulse() && pn.IsBefore(before)
 }
 
+func isTimePulseBeforeOrEq(pn pulse.Number, before pulse.Number) bool {
+	return pn.IsTimePulse() && pn.IsBeforeOrEq(before)
+}
+
+func isSpecialTimePulseBeforeOrEq(pn pulse.Number, before pulse.Number) bool {
```

Comment:
should be
isSpecialOrTimePulseBeforeOrEq

Reply:
I'll rename it in my PR